config/develop/iam/iam_service.json

```
@@ -1,8 +1,8 @@
 {
   "basePath": "http://localhost:5000/",
   "claimPath": "http://localhost:5000/claims/ensure",
-  "disableClaimCheck": false,
-  "allowDummyToken": false,
+  "disableClaimCheck": true,
+  "allowDummyToken": true,
```

Bitte rückgängig machen. 

In der Test Umgebung kann das gerne deaktiviert werden, dafür ist die da. Aber hier muss das an bleiben.

```suggestion
  "disableClaimCheck": false,
  "allowDummyToken": false,
```

---

Sry, hab ich vergessen wieder raus zu nehmen. Hast natürlich recht